ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -416,21 +341,157 @@
   (assert (some? (:beginning-of-week rw)) rw)
   (not (local-date-before? (:beginning-of-week rw) date)))
 
-(defn filter-by-vals [pred map]
-  (into {} (for [[k v] map
-                 :when (pred v)]
-             [k v])))
+(defn- remove-starting-nt-first
```

Comment:
replace would be better than remove

Reply:
True, renamed it a bit later. Will push.